dracor.odd

```diff
@@ -84,16 +86,17 @@
                         al. 2019</ref>), understood as corpora that expose an open, transparently
                     documented and (at least partly) research-driven API to make texts
                     machine-actionable. (cf. <ref target="#boerner_trilcke_2023">Börner and Trilcke
-                        2023</ref>). The documentation of the DraCor API can be accessed at ; <ref
-                        target="https://dracor.org/doc/api">https://dracor.org/doc/api</ref>.</p>
-                <p> DraCor uses a TEI customization that contains only selected TEI elements
-                    supported by the wider DraCor system and restricts the use of XML attributes and
-                    its values (see <ref target="#div_schema">Schema</ref>). The customization is
-                    documented in this <ref
-                        target="https://github.com/dracor-org/dracor-schema/blob/main/odd/dracor.odd"
-                        >ODD</ref> from which a RelaxNG schema (<ref
+                        2023</ref>). The documentation of the DraCor API can be accessed at <ref
+                        target="https://dracor.org/doc/api">https://dracor.org/doc/api</ref>. For
+                    additional information on the API and a documetation of the attributes returned
+                    in the JSON responses see section <ref target="#div_dracor-api">API</ref>.</p>
```

Comment:
OK, I fixed the broken target, and, OK, I can the underscores to dashes

Reply:
OK, done.  I think I changed all section identifiers to use dash only. I kept the underscores in the identifiers of the features though, e.g. `play_id`, `corpus_name` because these are the local identifiers I will create the URIs of the concept in the ontology from